ios/FluentUI/Card Nudge/CardNudgeTokens.swift

```diff
@@ -17,54 +17,34 @@ import SwiftUI
 /// Design token set for the `CardNudge` control.
 public class CardNudgeTokens: ControlTokens {
     /// Creates an instance of `CardNudgeTokens`.
-    convenience public init(style: MSFCardNudgeStyle) {
-        self.init(style: style,
-                  accentColor: nil,
-                  accentIconSize: nil,
-                  accentPadding: nil,
-                  backgroundColor: nil,
-                  buttonBackgroundColor: nil,
-                  buttonInnerPaddingHorizontal: nil,
-                  circleRadius: nil,
-                  circleSize: nil,
-                  cornerRadius: nil,
-                  horizontalPadding: nil,
-                  iconSize: nil,
-                  interTextVerticalPadding: nil,
-                  mainContentVerticalPadding: nil,
-                  minimumHeight: nil,
-                  outlineColor: nil,
-                  outlineWidth: nil,
-                  subtitleTextColor: nil,
-                  textColor: nil,
-                  verticalPadding: nil
-        )
+    public init(style: MSFCardNudgeStyle) {
+        self.style = style
+        super.init()
     }
 
     /// Creates an instance of `CardNudgeTokens` with optional token value overrides.
-    public init(style: MSFCardNudgeStyle,
-                accentColor: DynamicColor? = nil,
-                accentIconSize: CGFloat? = nil,
-                accentPadding: CGFloat? = nil,
-                backgroundColor: DynamicColor? = nil,
-                buttonBackgroundColor: DynamicColor? = nil,
-                buttonInnerPaddingHorizontal: CGFloat? = nil,
-                circleRadius: CGFloat? = nil,
-                circleSize: CGFloat? = nil,
-                cornerRadius: CGFloat? = nil,
-                horizontalPadding: CGFloat? = nil,
-                iconSize: CGFloat? = nil,
-                interTextVerticalPadding: CGFloat? = nil,
-                mainContentVerticalPadding: CGFloat? = nil,
-                minimumHeight: CGFloat? = nil,
-                outlineColor: DynamicColor? = nil,
-                outlineWidth: CGFloat? = nil,
-                subtitleTextColor: DynamicColor? = nil,
-                textColor: DynamicColor? = nil,
-                verticalPadding: CGFloat? = nil) {
-
-        self.style = style
-        super.init()
+    convenience public init(style: MSFCardNudgeStyle,
```

Comment:
We still don't have a solution for overriding "style/size aware" values in this scenario, right?
Are you going to explore the closure approach we discussed?

Reply:
I explored it a bit and while it was functional, it was also a bit unwieldy. I'll continue to explore, along with drawing up a binary size comparison for e.g. `Button`.

Reply:
Sounds good! Can you create an ADO item to track this effort? I think it deserves its own ADO task as it may cause more changes throughout.

Reply:
Done! ADO:5727871

Reply:
Thanks! (Just removed the ADO number because of our policy of not publishing internal info in the public github.)